book/build.py

```diff
@@ -0,0 +1,54 @@
+import subprocess
+from pathlib import Path
+import urllib.parse
+
+
+SCRIPT_DIR = Path(__file__).parent
+
+
+def generate_example(filename, title, strip_assets):
+    path_in = SCRIPT_DIR.parent / "examples" / f"{filename}.nbt"
+    path_out = SCRIPT_DIR / "src" / f"example-{filename}.md"
+
+    code = []
+    with open(path_in, "r") as fin:
+        for line in fin:
+            if not strip_assets or "assert_eq" not in line:
+                code.append(line)
+
+    url = f"https://numbat.dev/?q={urllib.parse.quote_plus(''.join(code))}"
```

Comment:
Was about to comment that this doesn't work on Python 3.10 locally for me, but you already fixed that :+1:

Reply:
Yes, I forgot that this only works with newer versions.